scss/partials/_dialog.scss

```diff
@@ -216,3 +216,22 @@
     align-self: flex-start;
   }
 }
+
+.jsxc-searchresults-table {
+  max-height: 600px;
+  min-width: 400px;
+  overflow: scroll;
+  width: 400px;
+}
+
+.jsxc-searchresults-table thead tr th {
```

Comment:
In Sass you can nest different rules to write less code.

```
.jsxc-searchresults-table {
  // your rules

  th { // th should be enough as selector
    // your th rules
  }
```

Reply:
ok th and td now in the brackets of .jsxc-searchresults-table